plugins/available/fzf.plugin.bash

```diff
@@ -6,6 +6,8 @@ about-plugin 'load fzf, if you are using it'
 
 [ -f ~/.fzf.bash ] && source ~/.fzf.bash
 
+command -v fd &> /dev/null && export FZF_DEFAULT_COMMAND='fd --type f'
```

Comment:
I don't like the fact that this overrides any custom value for `FZF_DEFAULT_COMMAND` that the user might have set up. Personally, I use

```
export FZF_DEFAULT_COMMAND='ag -U --hidden --ignore .git -g ""'
```

This would be overwritten by your change...

Reply:
thanks for the excellent feedback and catching that. Let me know if you are happy or not with the new change!